TestingTool/task.py

```diff
@@ -7,4 +7,20 @@
 
 
 class Task(EmbeddedDocument):
-    pass  # I'm just a placeholder
+    name = StringField(max_length=128, required=True)
+    action = StringField(max_length=128, required=True)
+    args = DictField()
+    reverse_result = BooleanField(default=False)
+    result = EmbeddedDocumentField(Result)
+
+
+def run(self, browser_instance):
+    data = DictField()
+    result = BooleanField()
+    for key in ACTION_LIST:
+        if key == self.action:
+            result, data = ACTION_LIST[key](self.args, browser_instance)
```

Comment:
Aici ai for-ul este inutil, intrucat tu apelezi ACTION_LIST[key]() doar daca key == self.action. Nu ai putea sa apelezi ACTION_LIST[altceva]()?